src/components/list/OrderedList.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { OrderedList as ChakraOrderedList } from '@chakra-ui/react';
+
+interface ListProps {
+  variant?: 'sm' | 'md';
+  start?: number;
+}
+
+const OrderedList: FC<PropsWithChildren<ListProps>> = ({
+  children,
+  variant,
```

Comment:
Since `variant` is optional prop, we should set default value for it.

Reply:
@ilay149 you can do this the same way as we set a default for `start` the line below

Reply:
after further investigation, we don't want a default here as this adds spacing. We want to be able to pass nothing to have the component without spacing.